client/src/app/site/assignments/components/assignment-detail/assignment-detail.component.ts

```diff
@@ -196,10 +196,10 @@ export class AssignmentDetailComponent extends BaseViewComponent implements OnIn
             phase: null,
             tags_id: [],
             attachments_id: [],
-            title: '',
-            description: '',
-            poll_description_default: '',
-            open_posts: 0,
+            title: ['', Validators.required],
+            description: [''],
+            poll_description_default: [''],
+            open_posts: [0, Validators.required],
```

Comment:
`open_posts` seems to have no effect. The form accepts this field, even tho it's empty.